tests/python_client/testcases/test_search.py

```diff
@@ -2908,21 +2896,17 @@ def test_search_with_output_fields(self, nb, nq, dim, auto_id, _async):
         # 2. search
         log.info("test_search_with_output_fields: Searching collection %s" % collection_w.name)
         vectors = [[random.random() for _ in range(dim)] for _ in range(nq)]
-        res = collection_w.search(vectors[:nq], default_search_field,
-                                  default_search_params, default_limit,
-                                  default_search_exp, _async=_async,
-                                  output_fields=[default_int64_field_name,
-                                                 default_float_field_name],
-                                  check_task=CheckTasks.check_search_results,
-                                  check_items={"nq": nq,
-                                               "ids": insert_ids,
-                                               "limit": default_limit,
-                                               "_async": _async})[0]
-        if _async:
-            res.done()
-            res = res.result()
-        assert len(res[0][0].entity._row_data) != 0
-        assert (default_int64_field_name and default_float_field_name) in res[0][0].entity._row_data
+        output_fields = [default_int64_field_name, default_float_field_name]
```

Comment:
Suggest to add value check for output_fields.

Reply:
I will add the value check in the next pr.

Reply:
OK